ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/ImageLoader.qml

```diff
@@ -9,7 +9,7 @@ Item {
     property bool isCurrentUser: false
     property url source
     property bool playing: true
-    property bool isAnimated: !!source && source.toString().endsWith('.gif')
+    property bool isAnimated: true
```

Comment:
Is this a leftover of a test or we start it as true and change it later?

Reply:
It is changed later (when image is clicked), but it default to false for all images. According to the documentation, this is the default anyway, and the `AnimatedImage` control has the ability to handle static images on its own anyway. 

https://doc.qt.io/qt-5/qml-qtquick-animatedimage.html#playing-prop

Additionally, this allows us to support animated image types that aren't necessarily limited to images that end in a `.gif` extension.